Add tutorial steps for manage dependencies (#3)
AICoE/manage-dependencies-tutorial: docs/share-your-work.md

# Share your work

> At this step of the tutorial you learned how to work on your project safely, supported by bots and pipelines that make automatic release for you and maintain the dependencies in your project. This last step of the tutorial will show you how you can share snapshots of your project, not only through container images but allowing others to repeat your experiments and interact with it using [Project Meteor][1].

Reviewer: ```suggestion
At this step of the tutorial you learned how to work on your project safely, supported by bots and pipelines that make automatic release for you and maintain the dependencies in your project. This last step of the tutorial will show you how you can share snapshots of your project, not only through container images but allowing others to repeat your experiments and interact with it bby using [Project Meteor][1].
```

# Project Meteor

> [Project Meteor][1] is a combined effort across the AICoE to provide a single tool for data scientists and other users where they can interact with, explore and leverage all of our services, tools and technologies for developing intelligent applications. Project Meteor is deployed on [Operate First][2] and you can use it following this this [link](http://meteor-shower-aicoe-meteor.apps.zero.massopen.cloud/)

Reviewer: ```suggestion
[Project Meteor][1] is a combined effort across the AICoE team at Red Hat, to provide a single tool for data scientists and other users where they can interact with, explore and leverage all of our services, tools and technologies for developing intelligent applications. Project Meteor is deployed on [Operate First][2] and you can use it following this this [link](http://meteor-shower-aicoe-meteor.apps.zero.massopen.cloud/)
```

> 1. Enter the URL in [Project Meteor][1].

Reviewer: ```suggestion
1. Enter the URL to your Git repository in [Project Meteor][1].
```

<div style="text-align:center">
<img alt="Enter URL Project Meteor" src="https://raw.githubusercontent.com/AICoE/manage-dependencies-tutorial/master/docs/images/ProjectMeteorEnterURL.png">
</div>

[Project Meteor][1] on the background will create two images for you:

> 
> - Jupyter Book where you can read about your experiment.
> 
> - JupyterLab environment where you can run your experiment.
> 

Reviewer: ```suggestion

- JupyterBook where you can view easily shareable, and rendered high-quality static content from experiment. 

- A live JupyterLab environment where you can interact with your project and run your experiments.

```
